src/Infrastructure/Data/Seeder/MongoDbDataSeeder.cs

```diff
@@ -14,8 +15,11 @@ public class MongoDbDataSeeder : IDataSeeder
     private readonly ILogger<MongoDbDataSeeder> _logger;
     private readonly IEntitySeeder[] _entitySeeders;
 
-    public MongoDbDataSeeder(IApplicationDbContext context, ILogger<MongoDbDataSeeder> logger)
+    public MongoDbDataSeeder(IApplicationDbContext context, ILogger<MongoDbDataSeeder> logger, IServiceScopeFactory serviceScopeFactory)
```

Comment:
`IServiceScopeFactory serviceScopeFactory`-> `IEmployeeService employeeService`
Is it possible?

Or

Why do we need `employeeService` here?